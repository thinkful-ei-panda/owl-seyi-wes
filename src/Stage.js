import React from 'react';
import StageParticipant from './StageParticipant';

function Stage(props){
    return (
        <section className='stage'>
        {
        props.participants.map((arrEle,index) => <StageParticipant key={props.participants[index].id} participant={props.participants[index]} />)
        }
        </section>
        
    )
}
export default Stage