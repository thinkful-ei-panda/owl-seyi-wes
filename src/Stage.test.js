import React from 'react';
import ReactDOM from 'react-dom';
import Stage from './Stage';
import store from './store'

import renderer from 'react-test-renderer';

//<Stage participants={props.participants}/>
describe('Stage', () => {

    const props={
        participants:store.participants
    }
  it('renders without crashing', () => {

    const div = document.createElement('div');
    ReactDOM.render(<Stage {...props}/>, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the UI as expected', () => {
    const tree = renderer
      .create(<Stage {...props}/>)
      .toJSON();
    expect(tree).toMatchSnapshot();  
  });

});

