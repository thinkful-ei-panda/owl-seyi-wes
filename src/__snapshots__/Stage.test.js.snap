// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stage renders the UI as expected 1`] = `
<section
  className="stage"
>
  <div
    className="stage-participant-container"
  >
    <div>
      <div
        className="stage-name"
      >
        Koren Templeton
      </div>
    </div>
    <div>
      <img
        alt="avatar"
        src="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
      />
    </div>
  </div>
  <div
    className="stage-participant-container"
  >
    <div>
      <div
        className="stage-name"
      >
        Ashla Attwool
      </div>
    </div>
    <div>
      <img
        alt="avatar"
        src="https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1"
      />
    </div>
  </div>
</section>
`;
